JD-4-5-6/src/main/java/com/example/weekone/WeekOneApplication.java

```diff
@@ -0,0 +1,33 @@
+package com.example.weekone;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.ArrayList;
+
+@SpringBootApplication
+public class WeekOneApplication {
+
+    public static void main(String[] args) {
+        SpringApplication.run(WeekOneApplication.class, args);
+        fibonacci(0, 1, 13);
+    }
+
+    public static ArrayList fibonacci(int firstNumber, int secondNumber, int lengthOfSequence){
+        // Example: firstNumber = 0 secondNumber = 1 lengthOfSequence= 13
+        ArrayList result = new ArrayList();
+        for ( int i = 1; i <= lengthOfSequence; i++){
```

Comment:
Iterating an array that is non-zero index based is not standard in programming. Is there a reason you chose a `1` based index for the array?

Reply:
No, It should have been zero. Sorry for that. I updated it.